templateutils/funcs.go

```diff
@@ -198,11 +206,11 @@ func findField(fieldPath []string, v reflect.Value) reflect.Value {
 	return f
 }
 
-func filterField(obj interface{}, field, val string, cmp func(string, string) bool) (retval interface{}) {
+func filterField(fnname string, obj interface{}, field, val string, cmp func(string, string) bool) (retval interface{}) {
```

Comment:
No they don't.  @rbradford spotted a similar naming issue that was not detected by golint.  I'll  fix the names.  Thanks.